docs/sdks/sdk-ref-python.md

```diff
@@ -55,77 +65,93 @@ The package is published in this location:
-| `endpoint` | The endpoint for the EUID service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to EUID, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+| Role                                      | Create Instance of Class | Link to Instructions                                                                  |
+|:------------------------------------------|:-------------------------|:--------------------------------------------------------------------------------------|
+| Publisher                                 | `Uid2PublisherClient`    | [Usage for Publishers](#usage-for-publishers)                                         |
+| DSP                                       | `BidstreamClient`        | [Usage for DSPs](#usage-for-dsps)                                                     |
+| Sharer (not currently supported for EUID) | `SharingClient`          | Not applicable                                                                        |
 
-## Interface 
+You will need to provide the values necessary for the SDK to authenticate with the EUID service.
 
-The interface allows you to decrypt EUID advertising tokens and return the corresponding raw EUID. 
+| Parameter    | Description                                                                                |
+|:-------------|:-------------------------------------------------------------------------------------------|
+| `base_url`   | The endpoint for the EUID service. See [Environments](../getting-started/gs-environments). |
+| `auth_key`   | The API key. See [EUID Credentials](../getting-started/gs-credentials).                    |
+| `secret_key` | The client secret. See [EUID Credentials](../getting-started/gs-credentials).              |
 
->NOTE: When you use an SDK, you do not need to store or manage decryption keys.
+## Interface
+The `BidstreamClient` class allows you to decrypt EUID tokens into raw EUIDs.
 
-If you're a DSP, for bidding, call the interface to decrypt an EUID advertising token and return the EUID. For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
+For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
 
-The following is the decrypt method in Python:
+:::note
+When you use an SDK, you do not need to store or manage decryption keys.
+:::
 
-```python
-from uid2_client import EuidClientFactory
- 
-client = EuidClientFactory.create('https://prod.euid.eu', 'my-auth-token', 'my-secret-key')
-client.refresh_keys() # Note that refresh_keys() should be called once after create(), and then once per hour
-decrypted_token = client.decrypt(advertising_token)
-```
+### Decryption Response Content
 
-### Response Content
+Whether decrypting with the `BidstreamClient` class<!--  or the `SharingClient` class -->, the SDK returns the information shown in the following table.
```

Comment:
This needs to say "when" rather than "whether" which is used when there are two. Was this an error in the Java version?

When decrypting with the `BidstreamClient` class<!-- or the `SharingClient` class -->, the SDK returns the information shown in the following table.

Reply:
Yup, java version says the same https://euid.eu/docs/sdks/sdk-ref-java#decryption-response-content